easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'awscli'
+version = '1.16.290'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
```

Comment:
@timeu Any reason not to use `pip` here?

Reply:
No, will fix it. I copy and pasted the old verison and forgot it